connectors/s3-connector/src/test/scala/zio/connect/s3/S3ConnectorSpec.scala

```diff
@@ -222,6 +238,21 @@ trait S3ConnectorSpec extends ZIOSpecDefault {
           assertTrue(b1Objects == Chunk(key1)) &&
           assertTrue(b2Objects.sortBy(_.toString) == Chunk(key1, key2, key4).sortBy(_.toString))
 
+      },
+      test("moves preexisting object cross region") {
+        val bucket1 = BucketName(UUID.randomUUID().toString)
+        val bucket2 = BucketName(UUID.randomUUID().toString)
+        val key     = ObjectKey(UUID.randomUUID().toString)
+        for {
+          _        <- ZStream(bucket1) >>> createBucket()
+          _        <- ZStream(bucket2) >>> createBucket(usWest2Region)
+          content1 <- Random.nextBytes(5)
+          content2 <- Random.nextBytes(5)
+          _        <- ZStream.fromChunk(content1) >>> putObject(bucket1, key)
+          _        <- ZStream.fromChunk(content2) >>> putObject(bucket2, key, usWest2Region)
+          _        <- ZStream(S3Connector.MoveObject(bucket1, key, bucket2, key)) >>> moveObject(usEast1Region, usWest2Region)
+//          copiedContent <- getObject(bucket2, key, usWest2Region).runCollect
+        } yield assertTrue(true) //assertTrue(copiedContent == content1)
```

Comment:
I think we should still perform the commented test.

Reply:
Not sure how this was left commented, fixed